src/domains/vote/components/DelegateTable/DelegateTable.tsx

```diff
@@ -39,8 +38,9 @@ export const DelegateTable: FC<DelegateTableProperties> = ({
 
 	const columns = useDelegateTableColumns({ isLoading, network: selectedWallet.network() });
 
+	const delegatesPerPage = useMemo(() => (isMainsailNetwork(selectedWallet.network()) ? 53 : 51), [selectedWallet]);
```

Comment:
this should be taken from the network configuration instead, as that defines the active delegates without having to rely on explicit checks like this